### PR TITLE
BLIP - update license

### PR DESCRIPTION
**Pull request by sharma-riti**
BLIP sample model cards - 

Image To Text
[BLIP](https://ml.azure.com/model/Salesforce-BLIP-image-captioning-base:3/details?wsid=/subscriptions/381b38e9-9840-4719-a5a0-61d9585e1e91/resourceGroups/automlimage_eastus2_rg/providers/Microsoft.MachineLearningServices/workspaces/risha-ws-eastus2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#overview) - [Pipeline](https://ml.azure.com/experiments/id/06018dd1-a0cf-4712-97c0-6bf8b67e1a3e/runs/great_pencil_990pvcrkq0?wsid=/subscriptions/381b38e9-9840-4719-a5a0-61d9585e1e91/resourceGroups/automlimage_eastus2_rg/providers/Microsoft.MachineLearningServices/workspaces/risha-ws-eastus2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#) 
[BLIP2](https://ml.azure.com/model/Salesforce-BLIP-2-opt-2-7b-image-to-text:3/details?wsid=/subscriptions/381b38e9-9840-4719-a5a0-61d9585e1e91/resourceGroups/automlimage_eastus2_rg/providers/Microsoft.MachineLearningServices/workspaces/risha-ws-eastus2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#overview) - [Pipeline](https://ml.azure.com/experiments/id/e4c50711-8963-4ff0-9dc3-28f70aa77ad7/runs/sweet_pear_5g9vh019zf?wsid=/subscriptions/381b38e9-9840-4719-a5a0-61d9585e1e91/resourceGroups/automlimage_eastus2_rg/providers/Microsoft.MachineLearningServices/workspaces/risha-ws-eastus2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

VQA
[BLIP](https://ml.azure.com/model/Salesforce-BLIP-vqa-base:3/details?wsid=/subscriptions/381b38e9-9840-4719-a5a0-61d9585e1e91/resourceGroups/automlimage_eastus2_rg/providers/Microsoft.MachineLearningServices/workspaces/risha-ws-eastus2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#overview) - [Pipeline](https://ml.azure.com/experiments/id/ceee9ac9-dbc2-4295-a08e-26758b7e3810/runs/witty_lemon_qx2rrjwkj5?wsid=/subscriptions/381b38e9-9840-4719-a5a0-61d9585e1e91/resourceGroups/automlimage_eastus2_rg/providers/Microsoft.MachineLearningServices/workspaces/risha-ws-eastus2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)
[BLIP2](https://ml.azure.com/model/Salesforce-BLIP-2-opt-2-7b-vqa:4/details?wsid=/subscriptions/381b38e9-9840-4719-a5a0-61d9585e1e91/resourceGroups/automlimage_eastus2_rg/providers/Microsoft.MachineLearningServices/workspaces/risha-ws-eastus2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#overview) - [Pipeline](https://ml.azure.com/experiments/id/6ca13eb2-34c3-4a66-890e-c3272eab210d/runs/sleepy_lime_hj43qqbyl8?wsid=/subscriptions/381b38e9-9840-4719-a5a0-61d9585e1e91/resourceGroups/automlimage_eastus2_rg/providers/Microsoft.MachineLearningServices/workspaces/risha-ws-eastus2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

### CODE REVIEW

File: assets/models/system/salesforce-blip-image-captioning-base/spec.yaml [hunk 1]
```diff
@@ -19,6 +19,6 @@ properties:
     model_id: Salesforce/blip-image-captioning-base
 tags:
    Preview: ''
-   license: mit
+   license: bsd-3-clause
    task: image-to-text
-version: 2
```
Comment (PhaniShekhar) [inline assets/models/system/salesforce-blip-image-captioning-base/spec.yaml:24]: Why is the version reverted ?

**Review by PhaniShekhar — APPROVED** [review]

File: assets/models/system/salesforce-blip-image-captioning-base/spec.yaml (hunk 1 above)
Comment (PhaniShekhar) [reply assets/models/system/salesforce-blip-image-captioning-base/spec.yaml:24]: I believe, we cannot overwrite a version if the artifacts are updated.

Comment (sharma-riti) [reply assets/models/system/salesforce-blip-image-captioning-base/spec.yaml:24]: there was no version 1 for blip models... and will be removing version 2...

**Review by bhimar — APPROVED** [review]
pipeline link for blip2 image captioning is broken

**Pull request merged**